code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -1,28 +1,32 @@
 /obj/item/weapon/storage/bible
 	name = "bible"
-	desc = "Apply to head repeatedly."
-	icon_state ="bible"
+	desc = "Apply to person's head repeatedly."

```

Comment:
Correct me if I'm wrong but shouldn't this be

"Apply to **a** person's head repeatedly."